python/lsst/ts/externalscripts/maintel/tma/random_walk.py

```diff
@@ -262,14 +295,17 @@ async def random_walk_azel_by_time(self):
 
             t = Time.now().to_value("isot")
             self.log.info(
-                f"{t:25s}{step:10d}{current_az:10.2f}{new_az:10.2f}"
+                f"{t:25s}{counter:10d}{current_az:10.2f}{new_az:10.2f}"
```

Comment:
This format is useful when running on a Jupyter Notebook because the log messages do not display timestamps. It also displayed the data as a table. However, I agree that we are in a different context here and should use an appropriate format. I applied the suggested changes.

Reply:
I very strongly recommend fixing the notebooks to display the date in log messages. If configuring the logger to print to stdout you can specify the format. If printing SAL log messages you can print data.private_sndStamp (after converting to ISO format, if you like -- I usually don't bother because I care more about time differences, which I find easier to compute from floats).